build-source.sh

```diff
@@ -15,6 +15,21 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+# Accepts file paths. Print the first file that exists, or nothing if none exists.
+first_existing_file () {
+  while [ -n "$1" ]; do
+    if [ -e "$1" ]; then
+      echo "$1"
+      return
+    fi
+
+    shift
+  done
+}
+
+source_or_sourcedebian () {
+  first_existing_file "source/${1}" "source/debian/${1}"
```

Comment:
Shouldn't this be reversed, so that `debian/${1}` is looked for first, and the current setup is used as fallback?

Reply:
Updated as suggested.